frontend/src/components/navigation/Header/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Header() {
```

Comment:
How would this be incorporated into the actual app? (If you have any tutorial / link to refer to, that'd work.)

Reply:
Right now it's incorporated into LandingPage (left more details under that comment).